jobs/ci-kubernetes-e2e-gce-federation.env

```diff
@@ -8,7 +8,7 @@ KUBE_GCS_RELEASE_BUCKET=kubernetes-federation-release
 KUBE_GCS_DEV_RELEASE_BUCKET=kubernetes-federation-release
 KUBE_NODE_OS_DISTRIBUTION=debian
 
-GINKGO_TEST_ARGS=--ginkgo.focus=\[Feature:Federation\]
+GINKGO_TEST_ARGS=--ginkgo.focus=\[Feature:Federation\] --ginkgo.skip=\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]
```

Comment:
so, shall we skip NoCluster and Serial ones from all 3 jobs, i.e CRI, GCI & GCE?